source/fundamentals/data-formats/document-data-format-bson.txt

```diff
@@ -13,11 +13,82 @@ Document Data Format: BSON
 Overview
 --------
 
-- Definition of BSON format, where it's used, why it's used.
-- Link to more info on BSON
-- What information you can find in the following sections
+This guide explains the BSON data format, how MongoDB uses it, and how to
+install the BSON library independently of the MongoDB driver.
 
-Installation
-------------
+BSON Data Format
+----------------
 
-- Instructions on how to add the dependency with Maven or Gradle
+**BSON**, or Binary JSON, is the data format that MongoDB uses to organize
+and store data. This data format includes all JSON data structure types and
+adds support for binary types. BSON is a serialized form of JSON optimized for 
+scan speed and transmission over the network. For more information on the
```

Comment:
Where did you get this info about what it's optimized for?

Reply:
It's interesting that even our main documentation about BSON doesn't say anything about what it's optimized for, so perhaps we should avoid saying anything here.  It's certainly not clear to me how BSON is optimized for transmission over a network, given the lack of compression of even repeated field names.

Reply:
Info from https://en.wikipedia.org/wiki/BSON#Efficiency
Is the wikipedia entry inaccurate then? Doesn't the format at least provide scan-ability as the message byte length is always specified vs JSON in which you'd need to parse the entire message before detecting the end?
I can leave it out if this doesn't seem significant or explained accurately of course.

Reply:
Personally, I think it provided good context for the reader, but I removed mention of both scan speed and network transmission.